src/patterns/AccountPageHeader/AccountPageHeader.tsx

```diff
@@ -0,0 +1,111 @@
+import { ComponentProps, forwardRef } from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import Text from '../../components/Text/Text';
+import { TextVariants } from '../../components/Text';
+import { Divider } from '../../components/Divider';
+import Button from '../../components/Button/Button';
+import { ButtonVariants } from '../../components/Button/types';
+import { Icon, IconName } from '../../components/Icon';
+import IconButton from '../../components/IconButton/IconButton';
+
+export interface AccountPageHeaderProps extends ComponentProps<'div'> {
+  /** The main title displayed at the top of the account page */
+  title: string;
+  /** Optional subtitle text that appears below the main title */
+  subheader?: string;
+  /**
+   * Configuration for the primary action button
+   * @property {() => void} onClick - Function to call when the button is clicked
+   * @property {string} label - Text to display on the button
+   * @property {IconName} [icon] - Optional icon to display alongside the text
+   */
+  primaryActionButton?: {
+    onClick: () => void;
+    label: string;
+    icon?: IconName;
+  };
+  /** Optional text displayed above the title */
+  overline?: string;
+  /**
+   * Array of icon actions to display
+   * Each object should have an icon name and an action function
+   */
+  iconActions?: Array<{
+    icon: IconName;
+    action: () => void;
```

Comment:
why is it called `action` here but `onClick` above?  And aren't they buttons after all? `secondaryActionButtons`.  They should probably also take a label to attach to the Button `aria-label` prop since they won't have translatable text any where else.  Alternatively, you could just have a set of buttons, and just render them in that order that they're specified?  For example why these should be two different props instead of just `actionButtons: ` with a `isPrimary` variant or something?